src/boot/is_debug.c

```diff
@@ -1,5 +1,14 @@
 #include "global.h"
 
+typedef struct ISVDbg {
+    /* 0x00 */ u32 magic; // IS64
+    /* 0x04 */ u32 get;
+    /* 0x08 */ u8 unk_08[0x14 - 0x08];
+    /* 0x14 */ u32 put;
+    /* 0x18 */ u8 unk_18[0x20 - 0x18];
+    /* 0x20 */ u8 data[0x10000 - 0x20];
+} ISVDbg;
```

Comment:
```suggestion
typedef struct ISVDbg {
    /* 0x00 */ u32 magic; // IS64
    /* 0x04 */ u32 get;
    /* 0x08 */ u8 unk_08[0xC];
    /* 0x14 */ u32 put;
    /* 0x18 */ u8 unk_18[0x8];
    /* 0x20 */ u8 data[0x10000 - 0x20];
} ISVDbg; // size = 0x10000
```

Guess the subtraction could be removed from `data` as well, but it's interesting that the struct seems to be made to be exactly 64 KB

Reply:
I went ahead and did it for `data` as well